app/controllers/catalog_controller.rb

```diff
@@ -543,13 +543,11 @@ def svc_catalog_provision
       options[:header] = @right_cell_text
       options[:target_id] = st.id
       options[:target_kls] = st.class.name
-      options[:dialog_locals] = {
-        :api_submit_endpoint    => "/api/service_catalogs/#{st.service_template_catalog_id}/service_templates/#{st.id}",
-        :api_action             => "order",
-        :finish_submit_endpoint => svc_catalog_provision_finish_submit_endpoint,
-        :cancel_endpoint        => "/catalog/explorer"
-      }
-      dialog_initialize(ra, options)
```

Comment:
This means no `@edit`.

The templates still look at `@edit` and even `@edit[:explorer]` .. is that intentional?

Reply:
You're right, it's only for when using the old product setting. I'll update with your suggested change, thanks.